Containerfile-rc

```diff
@@ -0,0 +1,35 @@
+#Windows/Linux
+#FROM python:3.12.1-bookworm
+
+#Mac
+FROM amd64/python:3.12.1-bookworm
+
+#Mac M1
+#FROM arm64v8/python:latest
```

Comment:
We might consider trying multiple `FROM` statements to build multiple images, one for each build system we need to support. See documentation on multi-platform images: https://docs.docker.com/build/building/multi-platform/.